### PR TITLE
Fixed test cases issue

### PR DESCRIPTION
**Pull request by sanketbansal**
#### Changes proposed in this pull request:

- Changes made in test cases to remove the **subscribe undefined error** occuring in modified pages, resolved by adding RouterTestingModule and removing fake routes.
- Added MockWindow Service for twitterfeed component testcases.

- Link to live demo: http://pr-157-evalai.surge.sh

### CODE REVIEW

File: src/app/components/get-involved/get-involved.component.spec.ts [hunk 1]
```diff
@@ -24,8 +24,6 @@ describe('GetInvolvedComponent', () => {
         GlobalService,
```
Comment (lunayach) [inline src/app/components/get-involved/get-involved.component.spec.ts:26]: Line 16-18 could also be removed.

File: src/app/components/home/twitter-feed/twitter-feed.component.spec.ts [hunk 2]
```diff
@@ -3,15 +3,17 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
...
 describe('TwitterFeedComponent', () => {
   let component: TwitterFeedComponent;
   let fixture: ComponentFixture<TwitterFeedComponent>;
 
   beforeEach(async(() => {
+    const MOCK_SERVICE = new MockWindowService(null);
     TestBed.configureTestingModule({
       declarations: [ TwitterFeedComponent ],
-      providers: [ WindowService ],
+      providers: [ {provide: WindowService, useValue: MOCK_SERVICE } ],
```
Comment (lunayach) [inline src/app/components/home/twitter-feed/twitter-feed.component.spec.ts:16]: @sanketbansal could we add some more specs (other than the default ones) here now, or you'll add them in the other already opened PR?

Comment (sanketbansal) [reply src/app/components/home/twitter-feed/twitter-feed.component.spec.ts:16]: I will add test cases in other open PRs. I created this PR as other PR are dependent over this PR.

File: src/app/components/get-involved/get-involved.component.spec.ts (hunk 1 above)
Comment (sanketbansal) [reply src/app/components/get-involved/get-involved.component.spec.ts:26]: Ok will do that

**Comment by sanketbansal** [comment]
@lunayach @Shekharrajak  please review

**Comment by lunayach** [comment]
@sanketbansal which tests does this PR help in passing?
It modifies some test files whose specs are already passing.

**Comment by sanketbansal** [comment]
Actually, I modified test cases because these were failing for modifications made in the components.
i.e #140, #141, #139

**Comment by lunayach** [comment]
But I see tests passing in those PR's currently. So, are those PR's inheriting these changes?

**Comment by sanketbansal** [comment]
yes, I have made these changes in those PRs so that they do not fail on TravisCI

**Comment by lunayach** [comment]
So, can we close this PR?

**Comment by sanketbansal** [comment]
Yes, If you do not have the problem of these changes showing in different PRs

**Comment by lunayach** [comment]
Okay, then let's take @RishabhJain2018's take on this.

**Comment by RishabhJain2018** [comment]
sure, we can close this PR.

**Pull request closed without merging**